app/Livewire/Server/ValidateAndInstall.php

```diff
@@ -159,7 +159,7 @@ public function validateDockerVersion()
                 $this->dispatch('refreshBoardingIndex');
                 $this->dispatch('success', 'Server validated.');
             } else {
-                $this->error = 'Docker Engine version is not 22+. Please install Docker manually before continuing: <a target="_blank" class="underline" href="https://docs.docker.com/engine/install/#server">documentation</a>.';
+                $this->error = 'Docker Engine version is not 25+. Please install Docker manually before continuing: <a target="_blank" class="underline" href="https://docs.docker.com/engine/install/#server">documentation</a>.';
```

Comment:
If reusing the configuration for docker version to install is a good approach here, instead of hard coded values, can I reuse it in this message but rephrase as:
> Minimum Docker Engine version 26 is not instaled. Pleas install Docker manually (...)

Reply:
Yes you can reuse this message there